Documentation/network/concepts/masquerading.rst

```diff
@@ -79,7 +79,8 @@ The eBPF-based masquerading can masquerade packets of the following IPv4 L4 prot
 
 By default, all packets from a pod destined to an IP address outside of the
 ``ipv4-native-routing-cidr`` range are masqueraded, except for packets destined
-to other cluster nodes. The exclusion CIDR is shown in the above output of
+to other cluster nodes (similarly with ``ipv6-native-routing-cidr`` for IPv6).
```

Comment:
obscure grammar nit, you could also go with "as with"
```suggestion
to other cluster nodes (similar to ``ipv6-native-routing-cidr`` for IPv6).
```

Reply:
Fixed, thank you